frontend/app/components/input-option.tsx

```diff
@@ -0,0 +1,16 @@
+import { type ComponentProps } from 'react';
+
+export interface InputOptionProps extends ComponentProps<'option'> {
+  label: string;
+  value: string;
+}
+
+export function InputOption(props: InputOptionProps) {
+  const { label, value, ...restProps } = props;
+
+  return (
+    <option value={value} data-testid="input-radio" {...restProps}>
```

Comment:
- `data-testid` needs to be unique here
- this isn't a radio input 🤣

Maybe require a mandatory ID be passed in to the select component, and then pass that ID down to the option component and set the test id to `option-${id}` or something like that.

Reply:
I disagre... the `InputOption` component consumer can override the `data-testid` if he pass it as a prop.

Reply:
> I disagre... the `InputOption` component consumer can override the `data-testid` if he pass it as a prop. But th testid is wrong though. It should be `data-testid="input-option"`.

That was my assumption when creating this component. I will fix the `data-testid` value though.

Reply:
The wrapping component currently has no way to pass a test id down to the option component. And I still assert that these IDs need to be unique, even when the wrapping component omits the testid. This is true for both `<InputSelect>` and `<InputOption>`.

**Re: using the value as the id:** sure, that's okay.. but what if the value has spaces or other weird characters? Maybe you should hash the value? ex:

``` typescript
createHash('shake256', { outputLength: 4 }).update('value').digest('hex');
→ '457bdb6e'
```

Reply:
`<InputOption>` now accepts an id, which is used to construct a value for `data-testId`.

Reply:
Passing the parent ID down to the option isn't quite enough. What if you have 10 option components? (they all have the ID of the parent select component)

And yes, I acknowledge that passing the ID down was in fact my original suggestion.

Reply:
Disregard my last comment, I see how you're doing it. Will resolve this thread.